src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTable.scala

```diff
@@ -73,14 +72,19 @@ case class PipelineTable(
     this
   }
 
-  /**
-   * This EITHER appends/changes the spark overrides OR sets them. This can only set spark params if updates
-   * are not passed --> setting the spark conf is really mean to be private action
-   *
-   * @param updates spark conf updates
-   */
+  def mode: WriteMode = { // initialize to constructor value
+    if (!exists && _mode == WriteMode.merge) {
+      val onetimeModeChangeMsg = s"MODE CHANGED from MERGE to APPEND. Target ${tableFullName} does not exist, first write will be " +
+        s"performed as an append, subsequent writes will be written as merge to this target"
+      if (config.debugFlag) println(onetimeModeChangeMsg)
+      logger.log(Level.INFO, onetimeModeChangeMsg)
+      WriteMode.append
+    } else _mode
+  }
+
```

Comment:
Let's talk more about this. I don't know how to merge into a target that doesn't yet exist. This is only if the target doesn't exist, otherwise merge will be used.